packages/core/src/components/Card/styles.ts

```diff
@@ -49,6 +49,10 @@ export const styleSheetContent: StyleSheet = ({ color, pattern, ui, unit }) => (
     paddingBottom: unit * 1.5,
   },
 
+  side_full_size: {
```

Comment:
please update to `side_fullSize` - the `_` is used to signal a modifier, then camelCase naming continues